src/ctia/http/exceptions.clj

```diff
@@ -34,10 +34,24 @@
   (logging/log! :error e (ex-message e))
   (ex/response-validation-handler e data request))
 
+(defn es-ex-message
+  [e data request]
+  (str
```

Comment:
I would prefer that we log a map instead of concatenating a string like this, this will not show up great on Kibana